packages/next/src/utils.ts

```diff
@@ -1283,6 +1289,8 @@ export async function getPrerenderManifest(
             manifest.dynamicRoutes[lazyRoute].prefetchDataRoute;
           prefetchDataRouteRegex =
             manifest.dynamicRoutes[lazyRoute].prefetchDataRouteRegex;
+          fallbackStatus = manifest.dynamicRoutes[lazyRoute].fallbackStatus;
+          fallbackHeaders = manifest.dynamicRoutes[lazyRoute].fallbackHeaders;
```

Comment:
Did we need new fields instead of the existing initialHeaders and initialStatus?

Reply:
These are new fields for the `dynamicRoutes` part of the prerender manifest, and are already included in Next.js.